modules/core/role/modules/account_admin/farm_role_account_admin.routing.yml

```diff
@@ -0,0 +1,7 @@
+farm_role_account_admin.settings:
+  path: 'farm/settings/account-admin'
+  defaults:
+    _form: '\Drupal\farm_role_account_admin\Form\AccountAdminSettingsForm'
+    _title: 'Account Admin Role settings'
+  requirements:
+    _permission: 'administer farm settings'
```

Comment:
Hmm I wonder if this should be a different permission... as is, any account admin could come in and change this setting. Basically it doesn't prevent the recursive issue.

If we introduce a new permission that no standard/managed farmOS role provides, then this setting would be entirely under the control of the host/user 1. They could modify the setting OR provide a new role w/ this new permission that allows trusted users to modify this setting.

Reply:
Yes I was just commenting on that. I think that a new permission is the right idea. I'll add a commit, standby...

Reply:
https://github.com/farmOS/farmOS/pull/714/commits/e8f69d33b5a6034dad89f555e7a3f77579123c7d